README.md

```
@@ -102,17 +102,17 @@ To permanently delete all saved history:
 
 `cliphist` is actively being developed. Here is the planned roadmap:
 
-- [ ] **Phase 1 (Core Functionality):**
-    - Robust background agent.
-    - SQLite-based storage.
-    - `list`, `get`, `search`, and `clear` commands.
+- [x] **Phase 1 (Core Functionality):**
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)